src/infra/redis-stateful-set.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
@amerkul  StatefulSet is nice, but where is the PersistentVolumeClaim for Redis?